applications/serial_lte_modem/prj.conf

```diff
@@ -7,8 +7,7 @@
 CONFIG_LOG=y
 CONFIG_LOG_DEFAULT_LEVEL=3
 CONFIG_STACK_SENTINEL=y
-CONFIG_NEWLIB_LIBC=y
-CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_PICOLIBC=y
```

Comment:
This cannot be merged before we update our test environments.

Reply:
> This cannot be merged before we update our test environments.

SLM CI does not complete while the job was submitted yesterday.
Previous, completed with environmental problem (Zephyr SDK version)